src/components/Sum.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a, b }) {
```

Comment:
Nice, but if you used named export "export function Sum" no need to change test file.